modules/nf-core/glimpse/phase/main.nf

```diff
@@ -1,16 +1,14 @@
 process GLIMPSE_PHASE {
     tag "$meta.id"
-    label 'process_high'
+    label 'process_medium'
 
     conda "bioconda::glimpse-bio=1.1.1"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/glimpse-bio:1.1.1--h2ce4488_2':
         'quay.io/biocontainers/glimpse-bio:1.1.1--hce55b13_1' }"
 
     input:
-        tuple val(meta) , path(input), path(input_index), val(input_region), val(output_region), val(index_region), path(samples_file)
-        tuple val(meta2), path(reference), path(reference_index)
-        path(map)
+        tuple val(meta) , path(input), path(input_index), val(input_region), val(output_region), path(reference), path(reference_index), path(map), path(samples_file)
```

Comment:
Why are you reordering this? Normally, we like to keep the references separate from the samples (even the indices for any reference)

Reply:
I did this due to parallelization problems.
The linked conversation on slack is [here](https://nfcore.slack.com/archives/CJRH30T6V/p1677851174147299)
I need the right reference and right map to be use with their corresponding input.

Is their another ways of doing that ?

Reply:
True, maybe I should rename it for better comprehension ?